yt/visualization/plot_modifications.py

```diff
@@ -1659,15 +1665,12 @@ def __call__(self, plot):
                   plot.data.ds.domain_right_edge[yax]
         period_x = plot.data.ds.domain_width[xax]
         period_y = plot.data.ds.domain_width[yax]
-        particle_x, particle_y = self._enforce_periodic(reg[pt, field_x],
-                                                        reg[pt, field_y],
+        particle_x, particle_y = self._enforce_periodic(self.region[pt, field_x],
+                                                        self.region[pt, field_y],
                                                         x0, x1, period_x,
                                                         y0, y1, period_y)
         gg = ( ( particle_x >= x0 ) & ( particle_x <= x1 )
            &   ( particle_y >= y0 ) & ( particle_y <= y1 ) )
-        if self.minimum_mass is not None:
-            gg &= (reg[pt, "particle_mass"] >= self.minimum_mass)
-            if gg.sum() == 0: return
```

Comment:
Since I think we only want to deprecate `minimum_mass`, not render it non-functional, this code needs to be restored.